src/components/Editor.vue

```diff
@@ -497,6 +497,8 @@ export default {
 					logger.debug('onLoaded: Pushing local changes to server')
 					this.$queue.push(updateMessage)
 				}
+			} else {
+				this.setInitialYjsState(documentSource, { isRich: this.isRichEditor })
```

Comment:
Nitpicking, but any reason to use `isRich` instead of `isRichEditor` here? I always find it a bit confusing if variable names are changed when passed on, and in this case `isRichEditor` becomes `isRich` and later becomes `enableRichEditing`.